code/__DEFINES/abnormalities.dm

```diff
@@ -11,6 +11,19 @@
 #define ABNORMALITY_WORK_ATTACHMENT "Attachment"
 #define ABNORMALITY_WORK_REPRESSION "Repression"
 
+// Declares the Signals that should be listened for on Work Start/Attempt
+#define COMSIG_INSTINCT_WORK_ATTEMPTED "instinct_attempted"
+#define COMSIG_INSIGHT_WORK_ATTEMPTED "insight_attempted"
+#define COMSIG_ATTACHMENT_WORK_ATTEMPTED "attachment_attempted"
+#define COMSIG_REPRESSION_WORK_ATTEMPTED "repression_attempted"
+#define COMSIG_WORK_ATTEMPTED "work_attempted"
+/// Declares the Signals that should be listened for on Work Completion
+#define COMSIG_INSTINCT_WORK_COMPLETED "instinct_completed"
+#define COMSIG_INSIGHT_WORK_COMPLETED "insight_completed"
+#define COMSIG_ATTACHMENT_WORK_COMPLETED "attachment_completed"
+#define COMSIG_REPRESSION_WORK_COMPLETED "repression_completed"
+#define COMSIG_WORK_COMPLETED "work_completed"
```

Comment:
1. Why is it in abnormality defines instead of signals?
2. There's absolutely NO reason to make signals for each work type. You can already send work type as an argument.